scripts/g.extension/g.extension.py

```diff
@@ -1403,8 +1417,8 @@ def download_source_code_official_github(url, name, outdev, directory=None):
     """
     if not directory:
         directory = os.path.join(os.getcwd, name)
-    classchar = name.split(".", 1)[0]
-    moduleclass = expand_module_class_name(classchar)
+    # classchar = name.split(".", 1)[0]
+    # moduleclass = expand_module_class_name(classchar)
```

Comment:
I'm not sure what you mean by these commented out lines. Variables seems unused. Just delete?

Reply:
OK